### PR TITLE
Upgrade Julia to 0.5.0

### PR DESCRIPTION
**Pull request by abcsds**
Updated julia cask to version 0.5.0 which came out sep 19

### CODE REVIEW

File: Casks/julia.rb [hunk 1]
```diff
@@ -10,8 +10,9 @@
 
   depends_on macos: '>= :lion'
 
-  app "Julia-#{version}.app"
-  binary "#{appdir}/Julia-#{version}.app/Contents/Resources/julia/bin/julia"
+  # Version 0.5.0 has removed the last 0, for next versions it might be added
```
Comment (vitorgalvao) [inline Casks/julia.rb]: Please remove the comment.

Comment (vitorgalvao) [inline Casks/julia.rb]: Please remove the comment.

File: Casks/julia.rb [hunk 2]
```diff
@@ -10,8 +10,9 @@
...
+  app "Julia-#{version.sub(%r{\.\d+$}, '')}.app"
```
Comment (vitorgalvao) [inline Casks/julia.rb]: Use `#{version.major_minor}` instead.

File: Casks/julia.rb [hunk 3]
```diff
@@ -10,8 +10,9 @@
...
+  binary "#{appdir}/Julia-#{version.sub(%r{\.\d+$}, '')}.app/Contents/Resources/julia/bin/julia"
```
Comment (vitorgalvao) [inline Casks/julia.rb]: Use `#{version.major_minor}` instead.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by abcsds** [comment]
Ready,
Thanks for the tips :)

**Comment by vitorgalvao** [comment]
There are now conflicts with the branch.

**Review by vitorgalvao — APPROVED** [review]

**Comment by adidalal** [comment]
Thank you for the contribution. It needed some fixes, so they were made in caf836e0934f98a22970b2a00df93dff7fff911f. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**